app/javascript/chat/components/ChannelFilterButton.jsx

```diff
@@ -2,14 +2,34 @@ import { h } from 'preact';
 import PropTypes from 'prop-types';
 import { Button } from '@crayons';
 
+/**
+ * This component renders a button that is used for filtering chat channels.
+ *
+ * @param {object} props
+ * @param {string} props.type - The type of chat channel selected
+ * @param {string} props.name - Used for testing and is displayed to the user on the button
+ * @param {boolean} props.active - Should the button have the `active` CSS class`?
```

Comment:
think there's just a wee extra backtick here

```suggestion
 * @param {boolean} props.active - Should the button have the `active` CSS class?
```